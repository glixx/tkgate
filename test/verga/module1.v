module top;
   reg ck;

   always
     begin
	#20 ck = ~ck;
      end


   always @(ck) $display("ck=%b",ck);

   initial
	begin
	   ck = 1'b0;
	   #500 $finish;
	end

   foo f1;

endmodule

module foo;
   initial
	$display("hello from foo");

endmodule
