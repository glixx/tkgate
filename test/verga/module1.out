hello from foo
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
ck=1
ck=0
